### DISCUSSION TITLE
Unaccessible notification

### DISCUSSION THREAD

**Discussion by MatthieuScarset** [community/community#124098] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I apparently have a notification somewhere but I cannot access it, at all.

As you can see in the screenshot below, there is 1 out of 0 notification for my account :shrug: 

Can you please debug this and delete this notification I cannot dismiss myself?

Thank you

![image](https://github.com/community/community/assets/7369593/6ef1b678-bd88-49d5-b9cd-491159326437)

**Comment by ebndev** [comment, marked as answer]
Hi @MatthieuScarset, thanks for being a part of the GitHub Community 😄 

You can see [this recent discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions. 

The GitHub product team may not be able to respond to every feedback submission, but I will be noting this as a bug!
